frontend/components/vault/Vault.vue

```diff
@@ -30,7 +30,13 @@
                         <tr>
                             <td>Next price update</td>
                             <td>
-                                <div v-if="vaultTransaction.nextPriceChange" class="flex items-center space-x-1">
+                                <div
+                                    v-if="
+                                        vaultTransaction.nextPriceChange &&
+                                        vaultTransaction.nextPriceChange instanceof Date
```

Comment:
What else can it be besides `instanceof Date`?

According to the type it is only allowed to be `Date` anyway: https://github.com/sidestream-tech/unified-auctions-ui/blob/24a1932c9eb9e12c69f052a0b1788574cdda5833/core/src/types.ts#L375

Maybe you wanted to check it against `NaN` instead?

Reply:
Hmm, Honestly I do not know the type that was passed when the error occurred (as I could not reproduce it and it was not given in the issue). 

Therefore I went out on a limb and hoped that checking it against Date would be enough. In the end it is only a theoretical decision (as it should always be date anyway). If we had a way to reproduce the bug this would be helpful. Otherwise we could also go ahead and check it against date.

Reply:
> as I could not reproduce it and it was not given in the issue

Oh wow! Next time please raise this in the issue instead of submitting a PR which doesn't fix anything!

The `Invalid Date` is shown whenever a stablecoin vault is fetched (as indicated in the issue). This is where the invalid date is produced:
https://github.com/sidestream-tech/unified-auctions-ui/blob/c3c3cc4ed3fe7d7cf408ceca9fd44a4f9aec143b/core/src/oracles.ts#L116

Therefore, please check for NaN instead of the instanceof